hmrc-mongo-play-26/src/main/scala/uk/gov/hmrc/mongo/play/json/Codecs.scala

```diff
@@ -148,6 +150,25 @@ trait Codecs {
     val doc = bsonDocumentCodec.decode(new JsonReader(o2.toString), DecoderContext.builder.build)
     doc.get("tempKey")
   }
+
+  implicit class JsonOps[A: Writes](a: A) {
+    def asBson(legacyNumbers: Boolean = false): BsonValue = toBson(a, legacyNumbers)
+
+    def asDocument(legacyNumbers: Boolean = false): ScalaDocument = toBson(a, legacyNumbers).asDocument()
+  }
+
+  implicit class BsonOps(bs: BsonValue) {
+    def asJson[T](implicit fjs: Reads[T]): T = toJson(bs).as[T]
```

Comment:
I think the `asJson` are misnamed - they convert `Bson -> T` (with intermediate Json).
Can we call them `fromBson: Bson -> T` - this would them match `toBson: T -> Bson`